src/OpenSage.Game/Content/ContentManager.cs

```diff
@@ -157,15 +160,22 @@ internal void LoadIniFile(FileSystemEntry entry)
         internal FileSystemEntry GetMapEntry(string mapPath)
         {
             var normalizedPath = FileSystem.NormalizeFilePath(mapPath);
-            if (UserDataFileSystem != null && normalizedPath.StartsWith(UserDataFileSystem.RootDirectory))
+
+            if (SageGame >= SageGame.Cnc3)
```

Comment:
Wasn't the check in UserMapsFilesystem meant to prevent that check here?

Reply:
Yes, but since we use virtual paths for games after CNC3, the ` mapPath.Substring(UserDataFileSystem.RootDirectory.Length + 1)` is not needed anymore.
But probably it doesn't harm either, I'll try to remove this `if` check, and see how it runs.